bioconda_utils/bioconda_utils-requirements.txt

```diff
@@ -7,21 +7,21 @@ conda-forge-pinning=2024.06.01.05.54.15
 
 # basics
 python=3.10.*
```

Comment:
I'm too afraid to bump python to 3.12 ... but if someone says its a good idea, please let me know!